src/_posts/2020-05-05-what-is-software-modernisation.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+asset-type: post
+name: what-is-software-modernisation
+title: What is Software Modernisation and why do you want to do it?
+title-es: ¿Qué es Software Modernisation y por qué puedes necesitarlo?
```

Comment:
There was a conversation where we concluded that having the title and description in Spanish but the main content in English would lead to a poor user experience. Has that changed or should we remove `title-es` and `description-es`?

Reply:
Yes this is ok to have Spanish title and description but English content. Helena said that this is ok and was happy with this we captured it in basecamp along with some other decisions.
https://3.basecamp.com/4397637/buckets/15050224/messages/2629479324